src/Magnum/ImGuiIntegration/Test/ContextGLTest.cpp

```diff
@@ -291,6 +293,19 @@ void ContextGLTest::constructMove() {
     MAGNUM_VERIFY_NO_GL_ERROR();
 }
 
+void ContextGLTest::moveAssignEmpty() {
+    Context a{NoCreate};
+    Context b{{}};
+
+    /* Move from empty */
+    b = std::move(a);
+    CORRADE_COMPARE(b.context(), nullptr);
+
+    /* Move into empty */
+    b = std::move(a);
+    CORRADE_COMPARE(a.context(), nullptr);
```

Comment:
Maybe add a third variant, moving empty into empty? It won't trigger anything in the current code but better safe than sorry :D

Reply:
Sure